app/assets/stylesheets/report.scss

```diff
@@ -2,26 +2,34 @@
   background: #fff;
   border-radius: 2px;
   box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
-  display: block;
-  float: left;
-  margin: 1rem;
+  margin-bottom: 2rem;
+  margin-top: 2rem;
   min-height: 300px;
-  position: relative;
+  padding: 15px;
   transition: all 0.2s ease-in-out;
-  width: 100%;
-  
+
   &:hover {
     box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
   }
+
+  th {
+    white-space: nowrap;
+  }
+
+  td {
+    width: 100%;
+  }
 }
 
 .report-title {
-  padding: 0.5em;
   text-align: center;
+  border-top: 1px solid #000;
+  border-bottom: 1px solid #000;
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.